src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -500,10 +543,12 @@ export const getAuthenticatorVerifyButtonElements = (
 export const getAuthenticatorEnrollButtonElements = (
   authenticatorOptions: IdxOption[],
   step: string,
+  locale?: string,
```

Comment:
I'm not sure if this transformer util is the right place for `locale`. It smells just a little bit to have this passed down through all of these functions when the rendering layer (`AuthenticatorButton` component) has the locale context already.

I'm thinking of two possibilities:
* We could do the locale formatting of a timestamp in the component
* We can do locale formatting in another transformer layer like we handle translations, possibly even combined into a locale transformer layer

The latter feels like it might be unnecessary complexity, but it does fit the theme of localization for that transform.

Curious if you have thoughts. We can discuss live during work hours.